### DISCUSSION TITLE
Welcome to Celeste's Discussions page!

### DISCUSSION THREAD

**Discussion by hwittenborn** [hwittenborn/celeste#95] [Announcements]
## 👋 Welcome!
Hey! This is the place to be if you have any questions or general feedback/comments regarding Celeste. I haven't used GitHub Discussions with any of my projects before so this is definitely something I'm experimenting with, but I'm open to seeing how it works out :).

If you got anything feel free to leave it below or in a new thread. Glad to see you're checking out this area!

**Comment by Xubuntufan** [comment]
hi Hunter
I just came across your project. fantastic.
a request... can you add access to Koofr please? https://koofr.eu/ 
they have an additional encrypted setup  which they call "vault" : https://vault.koofr.net/   access only command line via rclone. so a GUI would be great ;)
thanks so much
X

**Comment by ghost** [comment]
Hello, would it be possible to add an option for the program to start with the computer, and another to limit the number of checks per unit of time? It doesn't make much sense to have to remember to open the program manually, and I noticed that it performs too many checks for modifications. Thank you for the excellent program!

**Reply by olivierfriard** [reply]
Hi,
maybe it is possible to trigger sync when file are modified/created/deleted by inotifywait ?

like in this bash command:

```
inotifywait -m -r /PATH_TO_SYNC | while read path action file; do
    echo "File $file in directory $path was modified"
done
```
